tests/ui/_cli.js

```diff
@@ -30,16 +30,22 @@ define({
         // If we weren't provided username and password, let's set a grep to avoid login tests
         if(args.u == undefined && args.p == undefined) {
             greps.push('requires-login');
+            console.log('No username (-u) and password (-p) provided.  Tests requiring login will be skipped.');
         }
 
         // Set a document for wiki testing
         if(args.wd == undefined) {
             greps.push('requires-doc');
+            console.log('No wiki document (-wd) provided.  Most wiki tests will be skipped.');
         }
 
         // Set the final GREP value
         args.grep = greps.length ? ('^(?!.*?\\[' + greps.join('|') + '\\])') : '';
 
+        if(args.grep) {
+            console.log('Command line arguments have forced a grep to skip tests:  ' + args.grep);
```

Comment:
Hm, I'm not sure I understand. Is there another way of phrasing this? What will be different from the point of view of the person running the test?